README.md

```diff
@@ -9,30 +10,35 @@ Blurp is a web application written in React + Express + MongoDB.
 
 To get developing, you'll want to clone this repository first. You'll also want to create your own working branch so that your `git push` doesn't overwrite the main branch.
 
-1. Go to your terminal and `cd` into a working folder you want to work in. 
+1. Go to your terminal and `cd` into a working folder you want to work in.
 
 2. Now run `git clone https://github.com/pdx-blurp/blurp-frontend.git`.
 
 3. Next, make sure that you're linked to this remote repo by running these two commands:
+
 ```
 git remote add origin https://github.com/pdx-blurp/blurp-frontend.git
 ```
+
 ```
 git remote set-url origin https://github.com/pdx-blurp/blurp-frontend.git
 ```
+
 4. Add a new working branch in your local and change to it: `git checkout -b <first name><last initial>/<purpose of branch>`. Ex: `danielt/fixing-stuff`
 
-Now, you should be all set up to start developing! 
-**TIP:** *When developing via the browser, use Incognito mode to avoid those nasty cookies!*
+Now, you should be all set up to start developing!
+**TIP:** _When developing via the browser, use Incognito mode to avoid those nasty cookies!_
 
 ## Running the Server
 
 We'll be using Docker Containers soon TM. For now, navigate to /ecomapper-app and do the following:
```

Comment:
Caught "ecomapper-app" here

Reply:
saw that and will remove the original change and change that to blurp-app